### PR TITLE
39: Add login page

### PR DESCRIPTION
**Pull request by VladTkach**
Added login component and clear useless styles

### CODE REVIEW

File: frontend/src/app/modules/authentication/authentication-routing.module.ts [hunk 1]
```diff
@@ -9,6 +10,8 @@ const routes: Routes = [
         component: AuthenticationPageComponent,
         children: [
             { path: 'registration', component: RegistrationComponent },
+            { path: 'login', component: LoginComponent },
+            { path: '**', component: LoginComponent },
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/authentication/authentication-routing.module.ts]: either redirect or set in all to /

Comment (VladTkach) [reply frontend/src/app/modules/authentication/authentication-routing.module.ts]: actually i didn't really understand, redirect to page not found or just set / when wrong route?

Comment (Anatolii-Ivanov) [reply frontend/src/app/modules/authentication/authentication-routing.module.ts]: Redirect to /login)
Or set LoginComponent to /

Comment (VladTkach) [reply frontend/src/app/modules/authentication/authentication-routing.module.ts]: got it, and implement this

**Review by serhiiyasenev — DISMISSED** [review]

**Review by IlliaLop — DISMISSED** [review]

**Comment by alinaSielina** [comment]
Looks good, but could you make a slight update to the code and use our custom buttons (`app-button`)?

**Review by alinaSielina — COMMENTED** [review]
Looks good, but could you make a slight update to the code and use our custom buttons (`app-button`)?

File: frontend/src/app/modules/authentication/registration/registration.component.html [hunk 2]
```diff
@@ -7,10 +7,7 @@ <h4>Sign Up</h4>
             <button>Sign Up with Google</button>
         </div>
 
-        <div class="barrier">
-            <hr>
-            <p class="barrier-text">or</p>
-        </div>
+        <app-barrier [text]="'or'" style=""></app-barrier>
```
Comment (serhiiyasenev) [inline frontend/src/app/modules/authentication/registration/registration.component.html]: `style=""`

just `""`?

Comment (VladTkach) [reply frontend/src/app/modules/authentication/registration/registration.component.html]: yea, missed it

**Review by serhiiyasenev — DISMISSED** [review]

**Review by alinaSielina — APPROVED** [review]

**Review by IlliaLop — APPROVED** [review]

**Pull request merged**